ios-trivia-game/ViewControllers/GameOptionsViewController.swift

```diff
@@ -33,3 +78,30 @@ class GameOptionsViewController: UIViewController {
     */
 
 }
+
+extension GameOptionsViewController: UIPickerViewDelegate, UIPickerViewDataSource {
+    
+    // The number of columns of data
+    func numberOfComponents(in pickerView: UIPickerView) -> Int {
+        return 1
+    }
+    
+    // The number of rows of data
+    func pickerView(_ pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
+        if pickerView.tag == 1 {
+            return categoryPickerData.count
+        } else {
+            return numOfQuestionsPickerData.count
+        }
+    }
+    
+    // The data to return for the row and component (column) that's being passed in
+    func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
+        if pickerView.tag == 1 {
+            return categoryPickerData[row]
+        } else {
```

Comment:
Updated code, and tag information is here:
<img width="872" alt="screen shot 2016-11-22 at 9 54 24 am" src="https://cloud.githubusercontent.com/assets/2025025/20535517/e7941334-b099-11e6-8cf1-ac7ff5a6bed6.png">

<img width="868" alt="screen shot 2016-11-22 at 9 54 40 am" src="https://cloud.githubusercontent.com/assets/2025025/20535524/ef4dd402-b099-11e6-8bd7-ee3f7e18d1ef.png">